sys/include/riotboot/flashwrite.h

```diff
@@ -128,6 +128,11 @@ typedef struct {
  */
 #define RIOTBOOT_FLASHWRITE_SKIPLEN     sizeof(RIOTBOOT_MAGIC)
 
+/**
+ * @brief Magic number used to invalidate a slot
+ */
+#define INVALIDATE_HDR                  0xAA
+
```

Comment:
No need to have this in the global namespace - just move it to `sys/riotboot/flashwrite.c`